Helper/Quote.php

```diff
@@ -72,8 +72,18 @@ public function __construct(
 
     public function sendQuote(\Magento\Quote\Model\Quote $quote, \Drip\Connect\Model\Configuration $config, string $action)
     {
+      $items = [];
+      foreach ($quote->getAllItems() as $item) {
+          $itemData = [
+            'item_id' => $item->getId(),
+            'product_id' => $item->getProduct()->getId(),
```

Comment:
Does this work? It would save a DB lookup if so.
```suggestion
            'product_id' => $item->getProductId(),
```

Reply:
Let me give that a try. I pulled what I have from some of our old code.